.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,21 @@
+resources:
```

Comment:
Can we move this file from the root of the repo and instead put it in the `build` folder?

Reply:
I left it in the root because if it is in the root we get Resource Authorization automatically:
https://github.com/Microsoft/vsts-agent/blob/master/docs/preview/yamlgettingstarted-authz.md

They do have plans on supporting directory structure for this though:
https://github.com/Microsoft/vsts-agent/pull/1430

Reply:
That is unfortunate.